mobile/lib/modules/home/ui/control_bottom_app_bar.dart

```diff
@@ -103,15 +103,25 @@ class ControlBottomAppBar extends ConsumerWidget {
                         context: context,
                         builder: (BuildContext context) {
                           return DeleteDialog(
-                            onDelete: onDelete!,
+                            onDelete: () => onDelete?.call(false),
                           );
                         },
                       );
                     } else {
-                      onDelete!();
+                      onDelete?.call(true);
                     }
                   }
                 : null,
+            onLongPressed: enabled
+                ? () => showDialog(

```

Comment:
This block is identical to the case a few lines above. Consider moving this to a helper function if this makes things better to reduce code duplication.